src/diff/index.js

```diff
@@ -367,22 +384,14 @@ function doRender(props, state, context) {
  * component check for error boundary behaviors
  */
 function catchErrorInComponent(error, component) {
-	// thrown Promises are meant to suspend...
-	let isSuspend = typeof error.then === 'function';
-	let suspendingComponent = component;
+	if (typeof error.then === 'function') {
```

Comment:
Is it safe to just use error.then here?

Reply:
99% safe. As far as I checked it didn‘t impact gzipped size. Let me double check

Reply:
To be honest we could just make `options.catchError(error)` and move this to debug all together. Would shave off significant bytes

Reply:
-13B on you. Thx! Will add a test later. Improving coverage is on the TODO list anyhow